torch/nn/modules/loss.py

```diff
@@ -1216,7 +1216,7 @@ class MultiLabelSoftMarginLoss(_WeightedLoss):
 
     Shape:
         - Input: :math:`(N, C)` where `N` is the batch size and `C` is the number of classes.
-        - Target: :math:`(N, C)`, label targets padded by -1 ensuring same shape as the input.
+        - Target: :math:`(N, C)`, label targets must have the same shape as the input.
```

Comment:
Commenting here for posterity: 

The `padded by -1` here seems to have been erroneously copied over from the docs for `MultiLabelMarginLoss` where each row of target is implicitly a list of class indices for the corresponding sample, padded by `-1` if the `num_labels_corresponding_to_sample < C`

https://github.com/pytorch/pytorch/blob/ee4b99cc3a24d2b6ef23bfeb9d4b4afc52e60354/torch/nn/modules/loss.py#L832

This fix is correct as for MultiLabelSoftMarginLoss, every class should be assigned a probability in the target.